### PR TITLE
feat(Storybook): inspection tool improvements

### PR DESCRIPTION
**Pull request by micieslak**
### What does the PR do

It iterates the inspection tool towards more usable tool:
- more meaningful item names
- no clipping of auxiliary lines
- better selection of parent item to draw auxiliary lines
- finding item on the left list by clicking
- proper handling of SpriteSequence

### Affected areas
Strybook's inspection tool

### Screenshot of functionality (including design for comparison)

Screenshot from 2023-06-20 11-28-58

### CODE REVIEW

**Review by caybro — APPROVED** [review]
LGTM

File: storybook/src/Storybook/InspectionItemsList.qml [hunk 1]
```diff
@@ -11,7 +11,9 @@ ListView {
 
     delegate: Text {
         width: ListView.view.width
-        height: 30
+        height: implicitHeight * 1.5
+
+        font.pixelSize: ApplicationWindow.window.font.pixelSize - 1
```
Comment (alexjba) [inline storybook/src/Storybook/InspectionItemsList.qml:16]: How can the `window` be null here? 🤔 

Sometimes when selecting an item to inspect there will be this error:
InspectionItemsList.qml:16: TypeError: Cannot read property 'font' of null

Go to ProfileDialogView -> Inspect -> Click on `MockNickname hasn't joined any communities yet`

**Review by alexjba — APPROVED** [review]
Nice! Found one minor error. But it works fine despite of it

File: storybook/src/Storybook/InspectionItemsList.qml (hunk 1 above)
Comment (micieslak) [reply storybook/src/Storybook/InspectionItemsList.qml]: Thank you for spotting that! Sometimes it's better to keep simple things simple ;)

**Pull request merged**